### ISSUE TITLE
Analyse drop-outs in the Adwords -> completion user journey

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#345]
Ref  #301.

The Adwords conversion rate for WTT, looking at a period from 25th May (when, according to that ticket, tracking was definitely in place), is 0.25%.

From my days as a web editor in the commercial sector, we used 2% as a baseline for a good conversion rate, and we were selling pricey purchases, so for something as simple and transaction-led as WTT, I think this is a concern.

WTT only has one goal - completion of a letter - and that seems correct for a single-aim website. Since we're tracking visitors to the confirmation 'success' page, the drop-outs between clicks and conversions could be in many places along the line: unconfirmed messages, people who are just checking their representatives but don't want to contact them (we do run lots of ads on the theme of 'who is my MP' etc), and so on.

To further complicate things, some, but not all of the WTT Adwords lead to landing pages on the mySociety site, which springboard users onto WTT, so there's a further opportunity for dropping out.

As a first step I think we should:
- isolate data for people arriving from Adwords ads (even better if we can do it per ad group, as that would help us weed out and pause any ad groups that aren't delivering 'qualified traffic' - ie people likely to convert)
- look at their path through the site
- find the most common points of drop-out 
- decide whether the issue is with the Ads, the landing page or WTT itself
- decide whether there are any other actions on the site that we might count as a conversion, eg, if we decide that looking up who your representatives are is a valid use of WTT, we could count 'people who search for 'who is my MP?' and get as far as inputting their postcode' as a success.
- If there are massive drop-outs between the landing page and WTT (but not when we link directly to the site) we should change all ad groups to point directly at the site.

I *think* I can do most of this but might need to call on @jacksonj04.

**Comment by MyfanwyNixon** [comment]
OK there's something here I don't understand.

When I look at [Analytics for WTT's Adwords](https://analytics.google.com/analytics/web/#report/advertising-adwords-campaigns/a660910w56920593p58054409/%3F_u.date00%3D20160525%26_u.date01%3D20160824%26_.goalOption%3D1%26_r.drilldown%3Danalytics.campaignCampaignId%3AWriteToThem%20%2F%207450479%26explorer-table-dataTable.sortColumnName%3Danalytics.adClicks%26explorer-table-dataTable.sortDescending%3Dtrue%26explorer-table.plotKeys%3D%5B%5D%26explorer-table.rowCount%3D50/), it tells me that there have been 1,626 conversions since May 25, and that the total cost of Adwords was £13,799.84.

Even with a poor exchange rate, that's not the $275.77 per click mentioned above (it's £8.49).

That's just looking at those who reached the completion page. We also have a goal set up for postcode entry, don't know if that's registered on the ads as a goal at all. If that is taken into consideration, it gives the number of conversions as 19,143, which works out at 72p per conversion.

Clearly I'm missing something. Would appreciate @jacksonj04 's opinion!

**Comment by jacksonj04** [comment]
After doing some digging, it seems that WTT does *not* have the AdWords tracking code installed anywhere. In other words, any conversions on WTT are *not* being reported to AdWords. I think what we're seeing is the cost of turning an advert from the WTT group into a conversion on one of TWFY, FMS or WDTK. This is because we have a single adwords account for all our sites, because Google AdWords grant. There is no way around this.

In short, what we see in AdWords will be very high because it's being wildly misreported.

The value reported in Analytics will be a lot closer to reality. If you take a look at [this report](https://analytics.google.com/analytics/web/#report/advertising-adwords-campaigns/a660910w56920593p58054409/%3F_u.date00%3D20160525%26_u.date01%3D20160824%26_.goalOption%3D1/) you can see it's far closer to the £8.50 per conversion you mention.

**Comment by MyfanwyNixon** [comment]
Thanks so much for delving into that Nick! Can we install the tracking code on WTT?
